### ISSUE TITLE
Can't find variable: cy

### ISSUE THREAD

**Issue by zackwang1218** [cytoscape/cytosnap#14]
Hi, I'm trying to runt the quick start example bug I hit this error:

```
info: ReferenceError: Can't find variable: cy
info: 
info:   undefined:7 in anonymous
info:   :17
```
Did I do anything wrong here?

Thanks

**Comment by zackwang1218** [comment]
I think the problem is here:

``` javascript
var evalling = page.evaluate(function(){
      if( window.layoutFunction ){ options.layout = layoutFunction(); }

      if( window.styleFunction ){ options.style = styleFunction(); }

      cy.style( options.style );

      cy.add( options.elements );

      cy.on('layoutstop', function(){
        alert('layoutstop');
      });

      cy.layout( options.layout );
    });
```

Phantom can't find what `cy` is

**Comment by zack-wang-infotrack** [comment]
I think the problem is this:
`return page.open( path.join(__dirname, './phantom/index.html') );`
I'm running this on windows, and this gives me the path like `D:\xx\xxxx`, but if you try to open local html in phantom, you need to use `file:///D:/xxx/xxx`. Other than this, it's all good.

**Comment by maxkfranz** [comment]
@zack-wang-infotrack Can you confirm that your prepending of `file://` to the URI fixes this issue for you on Windows?  Have you tested this change on Linux or Mac as well?

Aside: Node and Phantom don't tend to work all that well on Windows.  If you're running Windows, you may want to use a Linux VM.  Vagrant is nice for this.

**Comment by zack-wang-infotrack** [comment]
@maxkfranz Yes, after I changed the file path to windows style everything works fine on windows. I haven't test Linux and Mac, I think it won't work with this change.

**Comment by maxkfranz** [comment]
`return page.open( 'file://' + path.join(__dirname, './phantom/index.html') );` works on Mac.  Published as 1.1.2

**Comment by rura6502** [comment]
i meet same issue in 2.0.0 version on windows. how can i check this problem? ;(

**Comment by rura6502** [comment]
i tried to change this "page.open( 'file:///' + path.join(__dirname, './phantom/index.html').replace(/\\/gi, "/") );" and working on me. i am windows10 / visual code

**Comment by zack-wang-infotrack** [comment]
@rura6502 yes, looks like it's still not being solved on windows, we are doing the same as you. This is a node problem, `path.join(__dirname, './phantom/index.html')` returns ` D:\xxxxx\xxx\phantom\index.html`, tested node windows v6.11.3 and v8.5.0.

**Comment by maxkfranz** [comment]
@rura6502 @zack-wang-infotrack Is this on Windows 10?  I don't have access to a VM for 7 or 8 for testing.

**Comment by rura6502** [comment]
i am windows 10.

**Comment by zack-wang-infotrack** [comment]
I'm on win10 as well, not sure about other versions.

**Comment by maxkfranz** [comment]
I think that should do it.  It works on my Windows 10 VM.  Published as 2.0.1.
